pkg/apis/configuration/validation/virtualserver.go

```diff
@@ -1173,9 +1222,12 @@ func validateRegexPath(path string, fieldPath *field.Path) field.ErrorList {
 const (
 	pathFmt    = `/[^\s{};]*`
 	pathErrMsg = "must start with / and must not include any whitespace character, `{`, `}` or `;`"
+	grpcFmt    = `[^\s{};]*`
+	grpcErrMsg = "must not include any whitespace character, `{`, `}`, or `;`"
 )
 
 var pathRegexp = regexp.MustCompile("^" + pathFmt + "$")
+var grpcRegexp = regexp.MustCompile("^" + grpcFmt + "$")
```

Comment:
small suggestion: moving the const and var so that is right above `validateGrpcService` where it is used. We try to do that with all the other vars and consts in the validation package.